src/components/Photo.jsx

```diff
@@ -1,15 +1,19 @@
 import styles from '../styles/photo'
 
 import React from 'react'
+import { Link } from 'react-router'
 
-const STACK_FILES_DOWNLOAD_PATH = 'http://cozy.local:8080/files/download'
+import { STACK_FILES_DOWNLOAD_PATH } from '../constants/config'
 
-const Photo = ({ photo }) => {
+export const Photo = ({ photo, router }) => {
```

Comment:
is it a good thing to pass the whole router here as we only need the pathname ? Could it just be:
```jsx
<Photo photo={photo} parentPath={router.location.pathname} />
```
just asking :)

Also : in PhotoList.jsx, the Photo component is given a `key` attribute which seems to be unused.